erfa/tests/helper.py

```diff
@@ -6,7 +6,7 @@
 import sys
 import types
 import warnings
-from distutils.version import LooseVersion
+import packaging.version
```

Comment:
I thought this was no longer needed now?

Reply:
yep, sorry